### PR TITLE
Upgrade dependencies to pass 'npm audit'

### PR DESCRIPTION
**Pull request by janlazo**
- grunt
- jshint,jscs -> eslint (includes single -> double quote fixes), using polyjuice to convert configs
- karma (PhantomJS to Headless Chromium via puppeteer)

### CODE REVIEW

**Comment by janlazo** [comment]
```
=== npm audit security report ===


                                 Manual Review
             Some vulnerabilities require your attention to resolve

          Visit https://go.npm.me/audit-guide for additional guidance


  Moderate        Prototype Pollution

  Package         lodash

  Patched in      >=4.17.11

  Dependency of   dgeni-alive [dev]

  Path            dgeni-alive > lodash

  More info       https://nodesecurity.io/advisories/782


  Moderate        Prototype Pollution

  Package         lodash

  Patched in      >=4.17.11

  Dependency of   dgeni-alive [dev]

  Path            dgeni-alive > wiredep > lodash

  More info       https://nodesecurity.io/advisories/782


  Low             Prototype Pollution

  Package         lodash

  Patched in      >=4.17.5

  Dependency of   dgeni-alive [dev]

  Path            dgeni-alive > lodash

  More info       https://nodesecurity.io/advisories/577


  Low             Prototype Pollution

  Package         lodash

  Patched in      >=4.17.5

  Dependency of   dgeni-alive [dev]

  Path            dgeni-alive > wiredep > lodash

  More info       https://nodesecurity.io/advisories/577


  High            Open Redirect

  Package         ecstatic

  Patched in      >=2.2.2 <3.0.0 || >=3.3.2 <4.0.0 || >=4.1.2

  Dependency of   dgeni-alive [dev]

  Path            dgeni-alive > http-server > ecstatic

  More info       https://nodesecurity.io/advisories/830


  Moderate        Denial of Service

  Package         ecstatic

  Patched in      >=2.0.0

  Dependency of   dgeni-alive [dev]

  Path            dgeni-alive > http-server > ecstatic

  More info       https://nodesecurity.io/advisories/553


  Moderate        Regular Expression Denial of Service

  Package         marked

  Patched in      >=0.6.2

  Dependency of   dgeni-alive [dev]

  Path            dgeni-alive > dgeni-packages > marked

  More info       https://nodesecurity.io/advisories/812


  Moderate        Regular Expression Denial of Service

  Package         mime

  Patched in      >= 1.4.1 < 2.0.0 || >= 2.0.3

  Dependency of   dgeni-alive [dev]

  Path            dgeni-alive > mime

  More info       https://nodesecurity.io/advisories/535


  Low             Regular Expression Denial of Service

  Package         braces

  Patched in      >=2.3.1

  Dependency of   dgeni-alive [dev]

  Path            dgeni-alive > dgeni-packages > nunjucks > chokidar >
                  anymatch > micromatch > braces

  More info       https://nodesecurity.io/advisories/786

found 9 vulnerabilities (3 low, 5 moderate, 1 high) in 4971 scanned packages
  9 vulnerabilities require manual review. See the full report for details.
```

**Comment by ngehlert** [comment]
just out of interest whats up with the single quote/double quote stuff? 🤔

**Comment by rubenv** [comment]
My though exactly, why does a security fix reformat all code?

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+    ],
+    "space-unary-ops": [
+      2,
+      {
+        "words": false,
+        "nonwords": false
+      }
+    ],
+    "space-infix-ops": 2,
+    "quotes": [
```
Comment (janlazo) [inline .eslintrc.json]: polyjuice converted `validateQuoteMarks` to the `quotes` eslint rule via https://github.com/brenolf/polyjuice/blob/master/lib/dictionaries/jscs/validateQuoteMarks.js. I assumed that the code style preferred double quotes and Grunt required for `Gruntfile.js` so I ran the eslint fixer. I assumed wrong so I'll revert the code reformat and remove this rule.

**Pull request closed without merging**